### PR TITLE
Tweak Props

### PR DESCRIPTION
**Pull request by aviraxp**
The commits Seraph08 picked almost reverts everything RayManFX did to improve stability in this commit https://github.com/halogenOS/android_device_oneplus_oneplus2/commit/6db937c05b16942c30faca4fcc1cf5eaf371dbc7

### CODE REVIEW

**Comment by xdevs23** [comment]
The camera props are for 4k video recording my friend

**Comment by xdevs23** [comment]
I don't think that these changes should be done, I need further review.

**Comment by aviraxp** [comment]
OK, the reason I do such tweaks is that PCM offload is broken, and Seraph08 kind of messed up the props. Close this if don't need them.

**Comment by xdevs23** [comment]
Mmm ok, some things are ok, but some are not...

File: BoardConfig.mk [hunk 1]
```diff
@@ -1,216 +1,214 @@
...
+
+# CFlags
+BOARD_GLOBAL_CFLAGS += -DUSE_RIL_VERSION_11 -DCOMPAT_SENSORS_M
+
+# Charger
+BOARD_CHARGER_DISABLE_INIT_BLANK := true
+BOARD_HEALTHD_CUSTOM_CHARGER_RES := $(PLATFORM_PATH)/charger/images
+
+# CNE and DPM
+TARGET_LDPRELOAD := libNimsWrap.so
```
Comment (msfjarvis) [inline BoardConfig.mk:114]: Don't we generally define this in the `init.*.rc`'s ?

File: BoardConfig.mk [hunk 2]
```diff
@@ -1,216 +1,214 @@
...
+# SELinux
+include device/qcom/sepolicy/sepolicy.mk
+
+BOARD_SEPOLICY_DIRS += $(PLATFORM_PATH)/sepolicy
+
+# Time services
+BOARD_USES_QC_TIME_SERVICES := true
+
+# CM Hardware
+BOARD_HARDWARE_CLASS += $(PLATFORM_PATH)/cmhw
```
Comment (msfjarvis) [inline BoardConfig.mk:210]: @xdevs23 have we retained cmhw from XOS-6.0?

**Review by msfjarvis — COMMENTED** [review]
Might need changes, but looks good in general

File: BoardConfig.mk (hunk 2 above)
Comment (xdevs23) [reply BoardConfig.mk:210]: We don't even need BOARD_HARDWARE_CLASS, as we don't use cmhw like that.
We use my implementations (example: keydisabler)

File: system.prop [hunk 3]
```diff
@@ -8,18 +8,14 @@ ro.qc.sdk.audio.fluencetype=fluence
 ro.qc.sdk.audio.ssr=false
 persist.audio.ssr.3mic=false
 media.aac_51_output_enabled=true
-audio.offload.pcm.16bit.enable=true
-audio.offload.pcm.24bit.enable=true
+audio.offload.pcm.16bit.enable=false
+audio.offload.pcm.24bit.enable=false
```
Comment (xdevs23) [inline system.prop:12]: Keep this.

File: system.prop [hunk 4]
```diff
@@ -8,18 +8,14 @@ ro.qc.sdk.audio.fluencetype=fluence
...
 use.voice.path.for.pcm.voip=true
-audio.offload.multiple.enabled=true
+audio.offload.multiple.enabled=false
```
Comment (xdevs23) [inline system.prop:14]: Keep this.

File: system.prop [hunk 5]
```diff
@@ -8,18 +8,14 @@ ro.qc.sdk.audio.fluencetype=fluence
...
 audio.offload.gapless.enabled=true
 mm.enable.smoothstreaming=true
 audio.offload.buffer.size.kb=1024
-audio.dolby.ds2.enabled=true
-audio.dolby.ds2.hardbypass=true
-audio.offload.passthrough=false
```
Comment (xdevs23) [inline system.prop:20]: Keep this.

File: system.prop [hunk 6]
```diff
@@ -8,18 +8,14 @@ ro.qc.sdk.audio.fluencetype=fluence
...
 persist.dpm.feature=1
-tunnel.audio.encode=false
```
Comment (xdevs23) [inline system.prop:22]: Keep this.

File: system.prop [hunk 7]
```diff
@@ -34,6 +30,7 @@ ro.bluetooth.wipower=true
 camera.disable_zsl_mode=1
 persist.camera.HAL3.enabled=0
 persist.camera.cpp.duplication=false
+media.camera.ts.monotonic=0
```
Comment (xdevs23) [inline system.prop:33]: What's this for?

File: system.prop [hunk 8]
```diff
@@ -70,11 +67,6 @@ ro.pip.gated=0
 # Graphics
 persist.hwc.mdpcomp.enable=true
 dalvik.vm.heapsize=36m
-persist.mdpcomp.4k2kSplit=1
-persist.hwc.mdpcomp.maxpermixer=5
-persist.mdpcomp_perfhint=50
-debug.mdpcomp.logs=0
-persist.hwc.ptor.enable=true
```
Comment (xdevs23) [inline system.prop:77]: Don't keep these changes.

File: system.prop [hunk 9]
```diff
@@ -86,11 +78,6 @@ ro.opengles.version=196610
 vidc.debug.perf.mode=2
 vidc.enc.dcvs.extra-buff-count=2
 
-# Camera
-persist.camera.cpp.duplication=false
-camera.disable_zsl_mode=1
-media.camera.ts.monotonic=0
```
Comment (xdevs23) [inline system.prop:92]: Don't keep these changes.

File: system.prop [hunk 10]
```diff
@@ -114,6 +101,7 @@ persist.rild.nitz_short_ons_3=
 ril.subscription.types=NV,RUIM
 DEVICE_PROVISIONED=1
 telephony.lteOnGsmDevice=1
+telephony.lteOnCdmaDevice=1
```
Comment (xdevs23) [inline system.prop:109]: Good.

**Review by xdevs23 — CHANGES_REQUESTED** [review]
Don't keep BoardConfig.mk changes.

File: system.prop (hunk 7 above)
Comment (aviraxp) [reply system.prop:33]: https://github.com/halogenOS/android_device_oneplus_oneplus2/commit/f1bdc02b37c90d5deb1c2b5dd4f614bfd39890db

This commit brings this. I just merge the camera tweaks in props into one.

Comment (xdevs23) [reply system.prop:33]: Ahh, ok, got it now.

File: system.prop (hunk 9 above)
Comment (xdevs23) [reply system.prop:92]: **Edit:** didn't notice, that the props are defined above, so we can keep these changes.

File: system.prop (hunk 7 above)
Comment (xdevs23) [reply system.prop:33]: Sorry for confusion

**Comment by aviraxp** [comment]
Now OK?

**Comment by xdevs23** [comment]
Remove the BoardConfig changes pls...

**Comment by xdevs23** [comment]
I can do that too if you feel lazy

**Comment by aviraxp** [comment]
That boardconfig commits aren't done by me but grarak. I am lazy so you can do this now.....

**Comment by xdevs23** [comment]
I can't see the exact changes. git diff is fucked up.
I guess line endings are swapped.

**Comment by xdevs23** [comment]
nvm, I'll merge

**Comment by xdevs23** [comment]
Thanks for your contribution.

**Pull request merged**